tools/src/main/java/org/apache/kafka/tools/ConsumerPerformance.java

```diff
@@ -265,12 +262,7 @@ protected static class ConsumerPerfOptions extends CommandDefaultOptions {
 
         public ConsumerPerfOptions(String[] args) {
             super(args);
-            brokerListOpt = parser.accepts("broker-list", "DEPRECATED, use --bootstrap-server instead; ignored if --bootstrap-server is specified. The broker list string in the form HOST1:PORT1,HOST2:PORT2.")
```

Comment:
Please update e2e file https://github.com/apache/kafka/blob/b9fe9f532fe11c658d07b11543e874526cb837b8/tests/kafkatest/services/performance/consumer_performance.py#L113

set the `bootstrap-server` instead of `broker-list` when the version >= 4.0